crates/rooch-types/src/transaction/ethereum.rs

```diff
@@ -2,13 +2,12 @@
 // SPDX-License-Identifier: Apache-2.0
 
 use super::{authenticator::Authenticator, AbstractTransaction, AuthenticatorInfo};
-use crate::{
-    address::EthereumAddress,
-    crypto::{BuiltinScheme, EcdsaRecoverableRoochSignature, Signature},
-    error::RoochError,
-};
+use crate::{address::EthereumAddress, error::RoochError};
 use anyhow::Result;
-use ethers::utils::rlp::{Decodable, Rlp};
+use ethers::{
+    types::Address,
+    utils::rlp::{Decodable, Rlp},
+};
 use fastcrypto::{
     hash::Keccak256,
     secp256k1::recoverable::Secp256k1RecoverableSignature,
```

Comment:
Do we need to use `ether` crates for Ethereum signature and address, or keep using `Secp256k1RecoverableSignature` from fastcrypto library? @baichuan3 @jolestar

Reply:
Both are okay, but we need to ensure compatibility with the Ethereum signature.

Reply:
> Both are okay, but we need to ensure compatibility with the Ethereum signature.

I prefer the latter one, cause it's the same logics as in `ethereum_address.move`.